src/main/java/appeng/bootstrap/BlockRendering.java

```diff
@@ -110,13 +110,13 @@ void apply( FeatureFactory factory, Block block, Class<?> tileEntityClass )
 
 		if( this.modelCustomizer != null )
 		{
-			factory.modelOverrideComponent.addOverride( block.getRegistryName().getResourcePath(), this.modelCustomizer );
+			factory.addModelOverride( block.getRegistryName().getResourcePath(), this.modelCustomizer );
 		}
 		else if( block instanceof AEBaseTileBlock )
 		{
 			// This is a default rotating model if the base-block uses an AE tile entity which exposes UP/FRONT as
 			// extended props
-			factory.modelOverrideComponent.addOverride( block.getRegistryName().getResourcePath(), ( l, m ) -> new AutoRotatingModel( m ) );
+			factory.addModelOverride( block.getRegistryName().getResourcePath(), ( l, m ) -> new AutoRotatingModel( m ) );
 		}
 
 		// TODO : 1.12
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)